lib/generators/rolemodel/testing/rspec/rspec_generator.rb

```diff
@@ -26,6 +26,7 @@ def add_spec_files
         template 'support/capybara_drivers.rb', 'spec/support/capybara_drivers.rb'
         template 'support/capybara_testid.rb', 'spec/support/capybara_testid.rb'
         template 'support/helpers/test_element_helper.rb', 'spec/support/helpers/test_element_helper.rb'
+        template 'support/helpers/action_cable_helper.rb', 'spec/support/helpers/action_cable_helper.rb'
```

Comment:
@Jeremy-Walton any reason the templates directory structure doesn't match the destination structure (missing `spec`)?
Since these files aren't leveraging erb replacement, `copy_file 'spec/support/helpers/action_cable_helper.rb'` would suffice if the paths were the same.  Not a big deal, just curious if there's a reason or just how it's always been..

Reply:
Not really sure. I believe it's just how it's been. Probably could be cleaned up in the future.